about/policy/sgid/index.md

```diff
@@ -10,16 +10,33 @@ categories: []
 date: 2019-12-04 10:01:27 -0500
 ---
 
-The following applies to data in the internal SGID database.
+The following policies apply to data in the internal SGID Enterprise Geodatabase.
 
-## Adding New Tables
+## Adding New Datasets to the SGID
 {: .text-left }
 
-When adding new data to the SGID, the steps outlined in the [Porter](https://github.com/agrc/porter) project will be followed.
+We strive to achieve and maintain a high standard of quality for the SGID. One of the best ways to do this is making sure all datasets coming into the SGID are high-quality datasets. We use the following policies and procedures to guide this effort.
```

Comment:
the sentence below could be removed and the concept would still hold: 
"One of the best ways to do this is making sure all datasets coming into the SGID are high-quality datasets"